vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,159 +2,92 @@
+1. If you are running the previous micro frontend, click `Ctrl+C` to end the process
 
-    <my-entity-details service-url="%REACT_APP_SERVICE_URL%" id="1" />
+2. Start the conference-details MFE
+``` sh
+ent bundle run conference-details
+```
+When the run is complete, you should see the details MFE showing the information for the Conference with ID 1.
 
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
+``` html
+    <conference-details id="1" />
+```
 
 ### Keycloak Settings and Issues 
-1. Change Keycloak Dev Settings
-
-If you want to use another Keycloak installation or change the service-URL of your widget, change the parameters set in the `.env.local` file generated by the Entando Blueprint in the root folder of your widgets. For instance, `ui/widgets/YOUR-ENTITY-NAME/tableWidget/.env.local`
-
-By default these variables are set to:
+1. Change Development Settings
 
-    REACT_APP_SERVICE_URL=http://localhost:8081/services/YOUR-APPLICATION-NAME/api
-    REACT_APP_KEYCLOAK_URL=http://localhost:9080/auth
-    REACT_APP_KEYCLOAK_REALM=jhipster
-    REACT_APP_KEYCLOAK_CLIENT_ID=web_app
+   If you want to use another Keycloak installation, modify the reference in the Keycloak YAML file located in the `svc/` folder.
 
-2. By default, Docker Compose recreates the Keycloak container and resets the H2 database each time it is started. To persist Keycloak changes across restarts, you can make the following updates: 
-   * In your src/main/docker/keycloak.yml file, replace\
-   `
-   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',` with 
-   ```
-   '-Dkeycloak.migration.strategy=IGNORE_EXISTING',
-   ```   
-   * In the same file, add a persistent volume under `volumes`:
+2. In this blueprint generated project, Docker Compose persists Keycloak data across restarts by default. If you want your data to reset on restarts:
+   
+   * Edit the `svc/keycloak.yml` file. Replace `-Dkeycloak.migration.strategy=IGNORE_EXISTING`,
+   with the following: 
+   ``` yaml
+   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',
    ```
+ 
+   * In the same file, remove this persistent volume statement under `volumes`:
+   ``` yaml
    - ./keycloak-db:/opt/jboss/keycloak/standalone/data
    ```   
-   * Keycloak changes should now be persistent. You can reset your Keycloak database by emptying the `src/main/docker/keycloak-db` directory and restarting the container.
+   * Keycloak should now reset every time you restart.
+
+2. Error `User is not authenticated` : If you see this message after starting the MFEs, it is likely that your Keycloak application is not running or the `.env.local` file is not configured properly. Check if Keycloak is running; otherwise, update the configuration in `microservice/conference-x/.env.local`.
```

Comment:
would rephrase "if you see"; whether or not you see it is different from whether or not it's generated

Reply:
I think this is really down in the weeds... and a style issue

Reply:
kind of. more about precision